llvm/lib/Transforms/Scalar/DeadStoreElimination.cpp

```diff
@@ -2054,7 +2147,7 @@ struct DSEState {
     return false;
   }
 
-  bool removePartiallyOverlappedStores(InstOverlapIntervalsTy &IOL) {
+  bool removePartiallyOverlappedIntrinsicStores(InstOverlapIntervalsTy &IOL) {
```

Comment:
I don't think we need to change this function name.

Reply:
I'm not sure about that, I thought the name doesn't accurately convey what the function does, it works on store-like intrinsics not on `store` instructions but I'm happy to make the change if you feel strongly enough about it.